uniffi_bindgen/src/bindings/kotlin/gen_kotlin/callback_interface.rs

```diff
@@ -17,55 +15,21 @@ impl CallbackInterfaceCodeType {
     pub fn new(id: String) -> Self {
         Self { id }
     }
-
-    fn ffi_converter_name(&self, oracle: &dyn CodeOracle) -> String {
-        format!("FfiConverter{}", self.canonical_name(oracle))
-    }
 }
 
 impl CodeType for CallbackInterfaceCodeType {
     fn type_label(&self, oracle: &dyn CodeOracle) -> String {
         oracle.class_name(&self.id)
     }
 
-    fn canonical_name(&self, oracle: &dyn CodeOracle) -> String {
-        format!("CallbackInterface{}", self.type_label(oracle))
+    fn canonical_name(&self, _oracle: &dyn CodeOracle) -> String {
+        format!("Type{}", self.id)
```

Comment:
You've done this change in multiple places. Should the default implementation go live in the same place as the default implementation of `ffi_converter_name`?

Reply:
I personally think so, but wouldn't that make it mega-match style?  Let's discuss that one more in #1145.

Reply:
No, this is still on `CodeType`, but they're all the same implementation. For the types that need to vary, it's trivial for them to vary.